docs/octopus-rest-api/tentacle.exe-command-line/index.md

```diff
@@ -9,7 +9,7 @@ hideInThisSection: true
 
 ## Commands {#tentacleCommandLine-Commands}
 
-Tentacle supports the following commands:
+`tentacle` supports the following commands:
```

Comment:
this makes sense to me now - its done via the  CommandLineDocsUpdater [here](https://github.com/OctopusDeploy/CommandLineDocsUpdater/blob/master/source/CommandLineDocsUpdater/Documentation/IndexFile.cs#L40)

(though it does flag up that the server is showing `octopus.server.exe` [over here](https://github.com/OctopusDeploy/docs/blob/master/docs/octopus-rest-api/octopus.server.exe-command-line/index.md))